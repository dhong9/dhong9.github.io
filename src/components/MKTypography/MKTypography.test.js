import renderer from "react-test-renderer";
import MKTypography from "components/MKTypography";

// Mocks
jest.mock("components/MKTypography/MKTypographyRoot", () => {
    const { forwardRef } = jest.requireActual("react");
    return {
        __esModule: true,
        default: forwardRef(() => <p>Mock Typography</p>)
    };
});

jest.mock("@mui/material/styles", () => ({
    ...jest.requireActual("@mui/material/styles"),
    styled: jest.fn().mockReturnValue(jest.fn().mockReturnValue("mock-style")),
  }));

describe("MKTypography", () => {
    it("renders", () => {
        const component = renderer.create(<MKTypography />);
        let tree = component.toJSON();
        expect(tree).toMatchSnapshot();
    });
});