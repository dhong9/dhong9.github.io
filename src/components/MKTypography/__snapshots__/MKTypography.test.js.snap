// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`MKTypography renders 1`] = `
<p>
  Mock Typography
</p>
`;
